modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/exportimport/data/handler/FileEntryStagedModelDataHandler.java

```diff
@@ -329,7 +333,7 @@ protected void doExportStagedModel(
 		}
 
 		_exportMetaData(portletDataContext, fileEntryElement, fileEntry);
-
+		_exportFriendlyURLEntries(portletDataContext, fileEntry);
 		_exportAssetDisplayPage(portletDataContext, fileEntry);
```

Comment:
Unless there's a dependency order, sort these calls in the same way everywhere. See my other comment about this for the other occurence.